### PR TITLE
Copter: apply yaw expo to all modes

### PR DESCRIPTION
**Pull request by lthall**
This PR contains a few changes:
1. add yaw expo to all flight modes.
2. reduce pilot input dead zone default
3. increase autotune timeout

### CODE REVIEW

File: ArduCopter/config.h [hunk 1]
```diff
@@ -509,8 +509,8 @@
 //////////////////////////////////////////////////////////////////////////////
 // Stabilize Rate Control
 //
-#ifndef ROLL_PITCH_INPUT_MAX
- # define ROLL_PITCH_INPUT_MAX      4500            // roll, pitch input range
+#ifndef ROLL_PITCH_YAW_INPUT_MAX
+ # define ROLL_PITCH_YAW_INPUT_MAX      4500            // roll, pitch input range
```
Comment (OXINARF) [inline ArduCopter/config.h]: Change comment?

**Comment by OXINARF** [comment]
By the way, this failed the SITL testing so that needs to be checked before merging it.

**Comment by rmackay9** [comment]
I tested this in SITL and it seems fine so pushing to master.

**Comment by OXINARF** [comment]
@rmackay9 Like I said this failed SITL testing in Travis but you merged it anyway. That means that all our testing, both in master and in every PR is failing because of it. There's no point in having a test if we don't care about it.

Also I don't get how a parameter that applies yaw expo to all modes is named ACRO_Y_EXPO, has a description that says *Acro Yaw Expo* and it's variable is acro_y_expo. That will just lead to confusion.

**Comment by magicrub** [comment]
@OXINARF What was failing in SITL?

**Comment by rmackay9** [comment]
@OXINARF, sorry about that.  I did test it on my machine (and it worked) before merging it.  I'll sort it out asap.

**Pull request closed without merging**